src/main/java/mate/academy/springboot/datajpa/service/impl/CategoryServiceImpl.java

```diff
@@ -34,6 +32,6 @@ public void delete(Long id) {
 
     @Override
     public Category update(Category category) {
-        return categoryRepository.saveAndFlush(category);
+        return categoryRepository.save(category);
```

Comment:
Check if a category exists before updating OR delete this method at all and use the method `save` from the service